redaxo/src/core/lib/console/command_loader.php

```diff
@@ -14,6 +14,13 @@ class rex_console_command_loader implements CommandLoaderInterface
 
     public function __construct()
     {
+        $commands = [
+            'core:sql-table:generate-code' => rex_console_command_sql_table_generate_code::class,
```

Comment:
Sollen die Core-Commands den Core-Präfix bekommen, oder lieber weglassen?

Nachteil: längerer Command-Name (wobei man die ja auch abkürzen kann, solange sie eindeutig bleiben, also `c:...` etc.)

Vorteil: Schöne Gruppierung der Commands:

![screenshot 2017-10-22 18 31 06](https://user-images.githubusercontent.com/330436/31864044-432e177a-b757-11e7-88e3-ba6832a39ae7.png)

Reply:
Vllt nen rex prefix?

Reply:
Hmm, gefällt mir glaube ich weniger. rex ist für mich alles, nicht nur der core (sieht man schon an meinem `rex` alias, siehe screenshot ;))

Reply:
stimmt macht kein sinn wenn das command rex heißt.
Ich denke es ist auch nicht wichtig woher das command kommt. Namespace sollte ein begriff sein der die problemdomäne irgendwie beschreibt

Reply:
Wir hatten irgendwo mal festgehalten, dass Addons ihren Key als Namespace nutzen sollten.
Aber bin mir selbst da auch nicht ganz sicher, ob das wirklich so eine feste Regel sein sollte.
Wobei es oftmals wahrscheinlich so oder so ganz gut passt.

Bzgl. Core bin ich aber auch offen. Können wir meinetwegen weglassen. Hab ich richtig verstanden, dass du dafür also plädierst oder? Dann nehme ich den `core:`-Präfix weg.

Reply:
Eventuell wäre es gut, erst einmal ein Liste mit den zukünftigen Kommandos zu erstellen. Eventuell ergibt sich da ein einheitliches Schema.

Reply:
Die liste haben wir in https://github.com/redaxo/redaxo/issues/1114, ggf sollten wir dort mal für alle use-cases versuchen einen namen zu finden...

Reply:
ich glaube ich würde zunächst den core: prefix weglassen.

ggf. sollten wir die console auch erstmal als experimental markieren, sodass wir dort besser "falsche entscheidungen" überarbeiten können